docs/source/api/user-guide/transformations/aggregation-quantile.ipynb

```diff
@@ -1,374 +1,515 @@
+            "execution_count": 7,
+            "metadata": {},
+            "outputs": [
+                {
+                    "data": {
+                        "text/plain": [
+                            "[6.5125, 14.833333333333334, 28.225]"
+                        ]
+                    },
+                    "execution_count": 7,
+                    "metadata": {},
+                    "output_type": "execute_result"
+                }
+            ],
+            "source": [
+                "def make_hist_quantiles(alphas, d_in, d_out, num_bins=500):\n",
+                "\n",
+                "    edges = np.linspace(*bounds, num=num_bins + 1)\n",
+                "    bin_names = [str(i) for i in range(num_bins)]\n",
+                "\n",
+                "    def make_from_scale(scale):\n",
+                "        return (\n",
+                "            input_space >>\n",
+                "            dp.t.then_drop_null() >>\n",
+                "            dp.t.then_find_bin(edges=edges) >> # bin the data\n",
+                "            dp.t.then_index(bin_names, \"0\") >> # can be omitted. Just set TIA=\"usize\", categories=list(range(num_bins)) on next line:\n",
+                "            dp.t.then_count_by_categories(categories=bin_names, null_category=False) >>\n",
+                "            dp.m.then_laplace(scale) >>\n",
+                "            # we're really only interested in the function on this transformation- the domain and metric don't matter\n",
+                "            dp.t.make_cast_default(dp.vector_domain(dp.atom_domain(T=int)), dp.symmetric_distance(), TOA=float) >>\n",
+                "            dp.t.make_quantiles_from_counts(edges, alphas=alphas)\n",
+                "        )\n",
+                "\n",
+                "    return dp.binary_search_chain(make_from_scale, d_in, d_out)\n",
+                "\n",
+                "hist_quartiles_meas = make_hist_quantiles(quartiles, max_contributions, epsilon)\n",
+                "hist_quartiles_meas(data)\n"
+            ]
+        },
+        {
+            "attachments": {},
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "A drawback of using this algorithm is that it can be difficult to choose the number of bins.\n",
+                "\n",
+                "If the number of bins is chosen to be very small, then the postprocessor will need to sum fewer instances of noise before reaching the bin of interest, resulting in a better bin selection.\n",
+                "However, the bin will be wider, so there will be greater error when interpolating the final answer.\n",
+                "\n",
+                "If the number of bins is chosen to be very large, then the same holds in the other direction.\n",
+                "\n",
+                "Estimating quantiles via the next algorithm can help make choosing the number of bins less sensitive.\n"
```

Comment:
Not a comment about changes in this PR, but I do wonder when it would be better to provide a mechanism via the API, rather than finding an example.

Reply:
These would be better rolled up into an API.

Reply:
- Filed #2378